frontend/tests/controllers/ContestCloneTest.php

```diff
@@ -106,6 +106,41 @@ public function testCreateContestCloneWithTheSameAlias() {
             $this->assertEquals('aliasInUse', $e->getMessage());
         }
     }
+    /**
+     * A PHPUnit data provider for all the plagiarism threshold values in a
+     * contest.
+     *
+     * @return list<array{0:bool, 1:int}>
+     */
+    public function plagiarismThresholdProvider(): array {
+        return [
+            [true, 90],
+            [false, 0],
+        ];
+    }
+    /**
+     * Check if the plagiarism value is stored correctly in the database.
+     * @dataProvider plagiarismThresholdProvider
+     */
+    public function testToValidatePlagiarismThresholdValue(
+        bool $checkPlagiarism
+        int $plagiarismThresholdExpected
+    ) {
+        // Create a contest
+        $contestData = \OmegaUp\Test\Factories\Contest::createContest(
+            new \OmegaUp\Test\Factories\ContestParams([
+                'plagiarism_threshold' => $checkPlagiarism,
+            ]);
+        )
```

Comment:
Maybe you mean `checkPlagiarism` here:
```suggestion
        $contestData = \OmegaUp\Test\Factories\Contest::createContest(
            new \OmegaUp\Test\Factories\ContestParams([
                'checkPlagiarism' => $checkPlagiarism,
            ])
        );
```
because it is the argument you added in `ContestParams` class. 

Please notice the right place where semicolon should be.

Reply:
Thanks. I think I got confused.